background/logging.js

```diff
@@ -0,0 +1,32 @@
+require('app-module-path').addPath(__dirname + '../../app');
+
+var path = require('path');
+
+var nconf = require('nconf');
+var bunyan = require('bunyan');
+
+formatBunyanInfoLog = function() {};
+formatBunyanInfoLog.prototype.write = function (rec) {
+    console.log('[%s] %s: %s',
+        rec.time,
+        bunyan.nameFromLevel[rec.level],
+        rec.msg);
+};
+
+logger = bunyan.createLogger({
+    name: 'Bitcannon',
```

Comment:
What about something like this: https://github.com/aidanharris/bitcannon-web/commit/7df89c52a5588a25be30e4368c842e42cd78aebc

I've created a logging class and it's initialised with the thing it's logging e.g `new logger('webui');` and then this is printed in the output of the logging.